### ISSUE TITLE
Error: failed to fetch from registry: karma

### ISSUE THREAD

**Issue by SriniTechHub** [npm/npm#8189]
```
npm ERR! Error: failed to fetch from registry: karma
npm ERR!     at RegClient.<anonymous> (C:\Program Files\nodejs\node_modules\npm\node_modules\npm-registry-client\lib\get.js:140:18)
npm ERR!     at cb (C:\Program Files\nodejs\node_modules\npm\node_modules\npm-registry-client\lib\request.js:26:9)
npm ERR!     at RegClient.<anonymous> (C:\Program Files\nodejs\node_modules\npm\node_modules\npm-registry-client\lib\request.js:114:10)
npm ERR!     at cb (C:\Program Files\nodejs\node_modules\npm\node_modules\npm-registry-client\lib\request.js:124:9)
npm ERR!     at RegClient.<anonymous> (C:\Program Files\nodejs\node_modules\npm\node_modules\npm-registry-client\lib\request.js:188:20)
npm ERR!     at self.callback (C:\Program Files\nodejs\node_modules\npm\node_modules\request\request.js:123:22)
npm ERR!     at Request.EventEmitter.emit (events.js:117:20)
npm ERR!     at ClientRequest.self.clientErrorHandler (C:\Program Files\nodejs\node_modules\npm\node_modules\request\request.js:232:10)
npm ERR!     at ClientRequest.EventEmitter.emit (events.js:95:17)
npm ERR!     at Socket.socketErrorListener (http.js:1547:9)
npm ERR! If you need help, you may report this *entire* log,
npm ERR! including the npm and node versions, at:
npm ERR!     <http://github.com/npm/npm/issues>

npm ERR! System Windows_NT 6.1.7601
npm ERR! command "C:\\Program Files\\nodejs\\\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "install"
npm ERR! cwd C:\Users\XPRK441\Desktop\systems-management
npm ERR! node -v v0.10.26
npm ERR! npm -v 1.4.3
npm ERR!
npm ERR! Additional logging details can be found in:
npm ERR!     C:\Users\XPRK441\Desktop\systems-management\npm-debug.log
npm ERR! not ok code 0
```

**Comment by KenanY** [comment]
@SriniTechHub While this may just be an issue with your internet connection (proxy, firewall, etc.), I'd like to first suggest [upgrading](https://github.com/npm/npm/wiki/Troubleshooting#upgrading-on-windows) `npm` before we proceed any further. There have been numerous fixes to just about everything since the `1.4` series.

**Comment by ehsalazar** [comment]
We haven’t heard back and we’re trying to clean up some old issues.  If this is still a problem, can you please reply and let us know?  We’ll be happy to reopen if necessary.
